config/file_config.go

```diff
@@ -213,6 +214,16 @@ type CollectionConfig struct {
 	MaxAlloc            MemorySize `yaml:"MaxAlloc"`
 }
 
+type SmartWrapperOptions struct {
+	BasicStoreType    string   `yaml:"Type" default:"local"`
+	SpanChannelSize   int      `yaml:"SpanChannelSize" default:"100"`
+	StateTicker       Duration `yaml:"StateTicker" default:"1s"`
+	SendDelay         Duration `yaml:"SendDelay" default:"2s"`
+	TraceTimeout      Duration `yaml:"TraceTimeout" default:"60s"`
+	DecisionTimeout   Duration `yaml:"DecisionTimeout" default:"3s"`
+	MaxTraceRetention Duration `yaml:"MaxTraceRetention" default:"24h"`
```

Comment:
I think `MacTraceRetention` is only used in redis store?

Reply:
I was thinking it's potentially useful in some future store. Are you doing additional config work? If so, you could move it. I don't feel strongly, though.

Reply:
I see. That makes sense. I was working on adding additional Config so I moved it out. I will play with it more in my PR for making the redis to work with the injection framework. We can leave it as it is for this PR